tools/ci-ts/test-helpers/chainlinkClient.ts

```diff
@@ -0,0 +1,103 @@
+import { ethers } from 'ethers'
+import execa from 'execa'
+// TODO replace import with CL @types package
+// https://www.pivotaltracker.com/story/show/171715396
+import { JobSpec, JobRun } from '../../../operator_ui/@types/operator_ui'
+import path from 'path'
+import Dockerode from 'dockerode'
+
+const API_CREDENTIALS_PATH = '/run/secrets/apicredentials'
+const docker = new Dockerode()
+
+/**
+ * interface for the data describing the CL node's keys
+ */
+interface KeyInfo {
+  address: string
+  ethBalance: ethers.utils.BigNumber
+  linkBalance: ethers.utils.BigNumber
+}
+
+export default class ChainlinkClient {
```

Comment:
Could we get field visibility modifiers on all of these? Would be nice to know what's `public` and what's `private` at a glance.

Reply:
This is awesome though, looks like a nice abstraction